### PR TITLE
Split docker cli into separate package, deb packaging

### PR DESCRIPTION
**Pull request by dhiltgen**
This splits out the CLI into a discrete package and removes
the engine from the engine package.  Instead the engine
is initialized via a post-inst script using the new CLI UX.

Currently relies on https://github.com/docker/cli/pull/1260

### CODE REVIEW

File: deb/common/docker-ce.postinst [hunk 1]
```diff
@@ -7,6 +7,10 @@ case "$1" in
 			if ! getent group docker > /dev/null; then
 				groupadd --system docker
 			fi
+                        # TODO Needs upgrade vs. install logic handling here
+                        if ctr --namespace docker container info dockerd 2&>1  > /dev/null ; then
+                            docker engine init
+                        fi
```
Comment (dhiltgen) [inline deb/common/docker-ce.postinst]: This script is **far** too naive for the final product, but should be enough to get us bootstrapped.

**Comment by seemethere** [comment]
Updated the PR title to reflect this phase of the package restructuring

<details>
<summary>Bits that are included in the new `docker-ce-cli` package</summary>

```
drwxr-xr-x root/root         0 2018-08-02 22:25 ./
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/bin/
-rwxr-xr-x root/root  49755088 2018-08-02 22:25 ./usr/bin/docker
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/bash-completion/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/bash-completion/completions/
-rw-r--r-- root/root    107361 2018-08-02 22:22 ./usr/share/bash-completion/completions/docker
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/doc/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/doc/docker-ce-cli/
-rw-r--r-- root/root       156 2018-08-02 22:25 ./usr/share/doc/docker-ce-cli/changelog.Debian.gz
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/fish/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/fish/vendor_completions.d/
-rw-r--r-- root/root     50230 2018-07-06 17:14 ./usr/share/fish/vendor_completions.d/docker.fish
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/man/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/man/man1/
-rw-r--r-- root/root       383 2018-08-02 22:25 ./usr/share/man/man1/docker-attach.1.gz
-rw-r--r-- root/root      5581 2018-08-02 22:25 ./usr/share/man/man1/docker-build.1.gz
-rw-r--r-- root/root       319 2018-08-02 22:25 ./usr/share/man/man1/docker-checkpoint-create.1.gz
-rw-r--r-- root/root       275 2018-08-02 22:25 ./usr/share/man/man1/docker-checkpoint-ls.1.gz
-rw-r--r-- root/root       275 2018-08-02 22:25 ./usr/share/man/man1/docker-checkpoint-rm.1.gz
-rw-r--r-- root/root       228 2018-08-02 22:25 ./usr/share/man/man1/docker-checkpoint.1.gz
-rw-r--r-- root/root       420 2018-08-02 22:25 ./usr/share/man/man1/docker-commit.1.gz
-rw-r--r-- root/root       289 2018-08-02 22:25 ./usr/share/man/man1/docker-config-create.1.gz
-rw-r--r-- root/root       334 2018-08-02 22:25 ./usr/share/man/man1/docker-config-inspect.1.gz
-rw-r--r-- root/root       319 2018-08-02 22:25 ./usr/share/man/man1/docker-config-ls.1.gz
-rw-r--r-- root/root       236 2018-08-02 22:25 ./usr/share/man/man1/docker-config-rm.1.gz
-rw-r--r-- root/root       234 2018-08-02 22:25 ./usr/share/man/man1/docker-config.1.gz
-rw-r--r-- root/root      1680 2018-08-02 22:25 ./usr/share/man/man1/docker-container-attach.1.gz
-rw-r--r-- root/root      1024 2018-08-02 22:25 ./usr/share/man/man1/docker-container-commit.1.gz
-rw-r--r-- root/root      2834 2018-08-02 22:25 ./usr/share/man/man1/docker-container-cp.1.gz
-rw-r--r-- root/root      5035 2018-08-02 22:25 ./usr/share/man/man1/docker-container-create.1.gz
-rw-r--r-- root/root       651 2018-08-02 22:25 ./usr/share/man/man1/docker-container-diff.1.gz
-rw-r--r-- root/root       952 2018-08-02 22:25 ./usr/share/man/man1/docker-container-exec.1.gz
-rw-r--r-- root/root       562 2018-08-02 22:25 ./usr/share/man/man1/docker-container-export.1.gz
-rw-r--r-- root/root       327 2018-08-02 22:25 ./usr/share/man/man1/docker-container-inspect.1.gz
-rw-r--r-- root/root       341 2018-08-02 22:25 ./usr/share/man/man1/docker-container-kill.1.gz
-rw-r--r-- root/root      1371 2018-08-02 22:25 ./usr/share/man/man1/docker-container-logs.1.gz
-rw-r--r-- root/root      2061 2018-08-02 22:25 ./usr/share/man/man1/docker-container-ls.1.gz
-rw-r--r-- root/root       565 2018-08-02 22:25 ./usr/share/man/man1/docker-container-pause.1.gz
-rw-r--r-- root/root       606 2018-08-02 22:25 ./usr/share/man/man1/docker-container-port.1.gz
-rw-r--r-- root/root       311 2018-08-02 22:25 ./usr/share/man/man1/docker-container-prune.1.gz
-rw-r--r-- root/root       265 2018-08-02 22:25 ./usr/share/man/man1/docker-container-rename.1.gz
-rw-r--r-- root/root       297 2018-08-02 22:25 ./usr/share/man/man1/docker-container-restart.1.gz
-rw-r--r-- root/root       851 2018-08-02 22:25 ./usr/share/man/man1/docker-container-rm.1.gz
-rw-r--r-- root/root      2457 2018-08-02 22:25 ./usr/share/man/man1/docker-container-run.1.gz
-rw-r--r-- root/root       414 2018-08-02 22:25 ./usr/share/man/man1/docker-container-start.1.gz
-rw-r--r-- root/root      1325 2018-08-02 22:25 ./usr/share/man/man1/docker-container-stats.1.gz
-rw-r--r-- root/root       323 2018-08-02 22:25 ./usr/share/man/man1/docker-container-stop.1.gz
-rw-r--r-- root/root       457 2018-08-02 22:25 ./usr/share/man/man1/docker-container-top.1.gz
-rw-r--r-- root/root       407 2018-08-02 22:25 ./usr/share/man/man1/docker-container-unpause.1.gz
-rw-r--r-- root/root      1883 2018-08-02 22:25 ./usr/share/man/man1/docker-container-update.1.gz
-rw-r--r-- root/root       385 2018-08-02 22:25 ./usr/share/man/man1/docker-container-wait.1.gz
-rw-r--r-- root/root       324 2018-08-02 22:25 ./usr/share/man/man1/docker-container.1.gz
-rw-r--r-- root/root       379 2018-08-02 22:25 ./usr/share/man/man1/docker-cp.1.gz
-rw-r--r-- root/root      2368 2018-08-02 22:25 ./usr/share/man/man1/docker-create.1.gz
-rw-r--r-- root/root       499 2018-08-02 22:25 ./usr/share/man/man1/docker-deploy.1.gz
-rw-r--r-- root/root       269 2018-08-02 22:25 ./usr/share/man/man1/docker-diff.1.gz
-rw-r--r-- root/root       372 2018-08-02 22:25 ./usr/share/man/man1/docker-events.1.gz
-rw-r--r-- root/root       535 2018-08-02 22:25 ./usr/share/man/man1/docker-exec.1.gz
-rw-r--r-- root/root       305 2018-08-02 22:25 ./usr/share/man/man1/docker-export.1.gz
-rw-r--r-- root/root       373 2018-08-02 22:25 ./usr/share/man/man1/docker-history.1.gz
-rw-r--r-- root/root      1175 2018-08-02 22:25 ./usr/share/man/man1/docker-image-build.1.gz
-rw-r--r-- root/root      1293 2018-08-02 22:25 ./usr/share/man/man1/docker-image-history.1.gz
-rw-r--r-- root/root       769 2018-08-02 22:25 ./usr/share/man/man1/docker-image-import.1.gz
-rw-r--r-- root/root       299 2018-08-02 22:25 ./usr/share/man/man1/docker-image-inspect.1.gz
-rw-r--r-- root/root       666 2018-08-02 22:25 ./usr/share/man/man1/docker-image-load.1.gz
-rw-r--r-- root/root      2201 2018-08-02 22:25 ./usr/share/man/man1/docker-image-ls.1.gz
-rw-r--r-- root/root       337 2018-08-02 22:25 ./usr/share/man/man1/docker-image-prune.1.gz
-rw-r--r-- root/root      3183 2018-08-02 22:25 ./usr/share/man/man1/docker-image-pull.1.gz
-rw-r--r-- root/root       863 2018-08-02 22:25 ./usr/share/man/man1/docker-image-push.1.gz
-rw-r--r-- root/root       464 2018-08-02 22:25 ./usr/share/man/man1/docker-image-rm.1.gz
-rw-r--r-- root/root       559 2018-08-02 22:25 ./usr/share/man/man1/docker-image-save.1.gz
-rw-r--r-- root/root      1091 2018-08-02 22:25 ./usr/share/man/man1/docker-image-tag.1.gz
-rw-r--r-- root/root       269 2018-08-02 22:25 ./usr/share/man/man1/docker-image.1.gz
-rw-r--r-- root/root       423 2018-08-02 22:25 ./usr/share/man/man1/docker-images.1.gz
-rw-r--r-- root/root       357 2018-08-02 22:25 ./usr/share/man/man1/docker-import.1.gz
-rw-r--r-- root/root       288 2018-08-02 22:25 ./usr/share/man/man1/docker-info.1.gz
-rw-r--r-- root/root      2963 2018-08-02 22:25 ./usr/share/man/man1/docker-inspect.1.gz
-rw-r--r-- root/root       293 2018-08-02 22:25 ./usr/share/man/man1/docker-kill.1.gz
-rw-r--r-- root/root       311 2018-08-02 22:25 ./usr/share/man/man1/docker-load.1.gz
-rw-r--r-- root/root       906 2018-08-02 22:25 ./usr/share/man/man1/docker-login.1.gz
-rw-r--r-- root/root       456 2018-08-02 22:25 ./usr/share/man/man1/docker-logout.1.gz
-rw-r--r-- root/root       437 2018-08-02 22:25 ./usr/share/man/man1/docker-logs.1.gz
-rw-r--r-- root/root       331 2018-08-02 22:25 ./usr/share/man/man1/docker-manifest-annotate.1.gz
-rw-r--r-- root/root       327 2018-08-02 22:25 ./usr/share/man/man1/docker-manifest-create.1.gz
-rw-r--r-- root/root       340 2018-08-02 22:25 ./usr/share/man/man1/docker-manifest-inspect.1.gz
-rw-r--r-- root/root       309 2018-08-02 22:25 ./usr/share/man/man1/docker-manifest-push.1.gz
-rw-r--r-- root/root       431 2018-08-02 22:25 ./usr/share/man/man1/docker-manifest.1.gz
-rw-r--r-- root/root      1080 2018-08-02 22:25 ./usr/share/man/man1/docker-network-connect.1.gz
-rw-r--r-- root/root      3019 2018-08-02 22:25 ./usr/share/man/man1/docker-network-create.1.gz
-rw-r--r-- root/root       313 2018-08-02 22:25 ./usr/share/man/man1/docker-network-disconnect.1.gz
-rw-r--r-- root/root      2162 2018-08-02 22:25 ./usr/share/man/man1/docker-network-inspect.1.gz
-rw-r--r-- root/root      2163 2018-08-02 22:25 ./usr/share/man/man1/docker-network-ls.1.gz
-rw-r--r-- root/root       312 2018-08-02 22:25 ./usr/share/man/man1/docker-network-prune.1.gz
-rw-r--r-- root/root       564 2018-08-02 22:25 ./usr/share/man/man1/docker-network-rm.1.gz
-rw-r--r-- root/root       251 2018-08-02 22:25 ./usr/share/man/man1/docker-network.1.gz
-rw-r--r-- root/root       251 2018-08-02 22:25 ./usr/share/man/man1/docker-node-demote.1.gz
-rw-r--r-- root/root       336 2018-08-02 22:25 ./usr/share/man/man1/docker-node-inspect.1.gz
-rw-r--r-- root/root       327 2018-08-02 22:25 ./usr/share/man/man1/docker-node-ls.1.gz
-rw-r--r-- root/root       251 2018-08-02 22:25 ./usr/share/man/man1/docker-node-promote.1.gz
-rw-r--r-- root/root       404 2018-08-02 22:25 ./usr/share/man/man1/docker-node-ps.1.gz
-rw-r--r-- root/root       266 2018-08-02 22:25 ./usr/share/man/man1/docker-node-rm.1.gz
-rw-r--r-- root/root       345 2018-08-02 22:25 ./usr/share/man/man1/docker-node-update.1.gz
-rw-r--r-- root/root       250 2018-08-02 22:25 ./usr/share/man/man1/docker-node.1.gz
-rw-r--r-- root/root       265 2018-08-02 22:25 ./usr/share/man/man1/docker-pause.1.gz
-rw-r--r-- root/root       325 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-create.1.gz
-rw-r--r-- root/root       260 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-disable.1.gz
-rw-r--r-- root/root       264 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-enable.1.gz
-rw-r--r-- root/root       303 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-inspect.1.gz
-rw-r--r-- root/root       365 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-install.1.gz
-rw-r--r-- root/root       877 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-ls.1.gz
-rw-r--r-- root/root       279 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-push.1.gz
-rw-r--r-- root/root       272 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-rm.1.gz
-rw-r--r-- root/root       248 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-set.1.gz
-rw-r--r-- root/root       374 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin-upgrade.1.gz
-rw-r--r-- root/root       265 2018-08-02 22:25 ./usr/share/man/man1/docker-plugin.1.gz
-rw-r--r-- root/root       278 2018-08-02 22:25 ./usr/share/man/man1/docker-port.1.gz
-rw-r--r-- root/root       471 2018-08-02 22:25 ./usr/share/man/man1/docker-ps.1.gz
-rw-r--r-- root/root       380 2018-08-02 22:25 ./usr/share/man/man1/docker-pull.1.gz
-rw-r--r-- root/root       294 2018-08-02 22:25 ./usr/share/man/man1/docker-push.1.gz
-rw-r--r-- root/root       245 2018-08-02 22:25 ./usr/share/man/man1/docker-rename.1.gz
-rw-r--r-- root/root       301 2018-08-02 22:25 ./usr/share/man/man1/docker-restart.1.gz
-rw-r--r-- root/root       354 2018-08-02 22:25 ./usr/share/man/man1/docker-rm.1.gz
-rw-r--r-- root/root       307 2018-08-02 22:25 ./usr/share/man/man1/docker-rmi.1.gz
-rw-r--r-- root/root     16105 2018-08-02 22:25 ./usr/share/man/man1/docker-run.1.gz
-rw-r--r-- root/root       318 2018-08-02 22:25 ./usr/share/man/man1/docker-save.1.gz
-rw-r--r-- root/root       919 2018-08-02 22:25 ./usr/share/man/man1/docker-search.1.gz
-rw-r--r-- root/root       308 2018-08-02 22:25 ./usr/share/man/man1/docker-secret-create.1.gz
-rw-r--r-- root/root       334 2018-08-02 22:25 ./usr/share/man/man1/docker-secret-inspect.1.gz
-rw-r--r-- root/root       320 2018-08-02 22:25 ./usr/share/man/man1/docker-secret-ls.1.gz
-rw-r--r-- root/root       235 2018-08-02 22:25 ./usr/share/man/man1/docker-secret-rm.1.gz
-rw-r--r-- root/root       234 2018-08-02 22:25 ./usr/share/man/man1/docker-secret.1.gz
-rw-r--r-- root/root      1879 2018-08-02 22:25 ./usr/share/man/man1/docker-service-create.1.gz
-rw-r--r-- root/root       337 2018-08-02 22:25 ./usr/share/man/man1/docker-service-inspect.1.gz
-rw-r--r-- root/root       497 2018-08-02 22:25 ./usr/share/man/man1/docker-service-logs.1.gz
-rw-r--r-- root/root       321 2018-08-02 22:25 ./usr/share/man/man1/docker-service-ls.1.gz
-rw-r--r-- root/root       398 2018-08-02 22:25 ./usr/share/man/man1/docker-service-ps.1.gz
-rw-r--r-- root/root       238 2018-08-02 22:25 ./usr/share/man/man1/docker-service-rm.1.gz
-rw-r--r-- root/root       333 2018-08-02 22:25 ./usr/share/man/man1/docker-service-rollback.1.gz
-rw-r--r-- root/root       307 2018-08-02 22:25 ./usr/share/man/man1/docker-service-scale.1.gz
-rw-r--r-- root/root      1959 2018-08-02 22:25 ./usr/share/man/man1/docker-service-update.1.gz
-rw-r--r-- root/root       259 2018-08-02 22:25 ./usr/share/man/man1/docker-service.1.gz
-rw-r--r-- root/root       580 2018-08-02 22:25 ./usr/share/man/man1/docker-stack-deploy.1.gz
-rw-r--r-- root/root       381 2018-08-02 22:25 ./usr/share/man/man1/docker-stack-ls.1.gz
-rw-r--r-- root/root       474 2018-08-02 22:25 ./usr/share/man/man1/docker-stack-ps.1.gz
-rw-r--r-- root/root       343 2018-08-02 22:25 ./usr/share/man/man1/docker-stack-rm.1.gz
-rw-r--r-- root/root       437 2018-08-02 22:25 ./usr/share/man/man1/docker-stack-services.1.gz
-rw-r--r-- root/root       305 2018-08-02 22:25 ./usr/share/man/man1/docker-stack.1.gz
-rw-r--r-- root/root       424 2018-08-02 22:25 ./usr/share/man/man1/docker-start.1.gz
-rw-r--r-- root/root       421 2018-08-02 22:25 ./usr/share/man/man1/docker-stats.1.gz
-rw-r--r-- root/root       298 2018-08-02 22:25 ./usr/share/man/man1/docker-stop.1.gz
-rw-r--r-- root/root       516 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-ca.1.gz
-rw-r--r-- root/root       700 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-init.1.gz
-rw-r--r-- root/root       283 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-join-token.1.gz
-rw-r--r-- root/root       423 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-join.1.gz
-rw-r--r-- root/root       262 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-leave.1.gz
-rw-r--r-- root/root       270 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-unlock-key.1.gz
-rw-r--r-- root/root       213 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-unlock.1.gz
-rw-r--r-- root/root       503 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm-update.1.gz
-rw-r--r-- root/root       256 2018-08-02 22:25 ./usr/share/man/man1/docker-swarm.1.gz
-rw-r--r-- root/root       299 2018-08-02 22:25 ./usr/share/man/man1/docker-system-df.1.gz
-rw-r--r-- root/root      2657 2018-08-02 22:25 ./usr/share/man/man1/docker-system-events.1.gz
-rw-r--r-- root/root      2535 2018-08-02 22:25 ./usr/share/man/man1/docker-system-info.1.gz
-rw-r--r-- root/root       356 2018-08-02 22:25 ./usr/share/man/man1/docker-system-prune.1.gz
-rw-r--r-- root/root       236 2018-08-02 22:25 ./usr/share/man/man1/docker-system.1.gz
-rw-r--r-- root/root       267 2018-08-02 22:25 ./usr/share/man/man1/docker-tag.1.gz
-rw-r--r-- root/root       264 2018-08-02 22:25 ./usr/share/man/man1/docker-top.1.gz
-rw-r--r-- root/root       306 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-inspect.1.gz
-rw-r--r-- root/root       284 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-key-generate.1.gz
-rw-r--r-- root/root       276 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-key-load.1.gz
-rw-r--r-- root/root       250 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-key.1.gz
-rw-r--r-- root/root       271 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-revoke.1.gz
-rw-r--r-- root/root       247 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-sign.1.gz
-rw-r--r-- root/root       274 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-signer-add.1.gz
-rw-r--r-- root/root       297 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-signer-remove.1.gz
-rw-r--r-- root/root       255 2018-08-02 22:25 ./usr/share/man/man1/docker-trust-signer.1.gz
-rw-r--r-- root/root       248 2018-08-02 22:25 ./usr/share/man/man1/docker-trust.1.gz
-rw-r--r-- root/root       267 2018-08-02 22:25 ./usr/share/man/man1/docker-unpause.1.gz
-rw-r--r-- root/root       638 2018-08-02 22:25 ./usr/share/man/man1/docker-update.1.gz
-rw-r--r-- root/root       691 2018-08-02 22:25 ./usr/share/man/man1/docker-version.1.gz
-rw-r--r-- root/root       911 2018-08-02 22:25 ./usr/share/man/man1/docker-volume-create.1.gz
-rw-r--r-- root/root       467 2018-08-02 22:25 ./usr/share/man/man1/docker-volume-inspect.1.gz
-rw-r--r-- root/root       573 2018-08-02 22:25 ./usr/share/man/man1/docker-volume-ls.1.gz
-rw-r--r-- root/root       309 2018-08-02 22:25 ./usr/share/man/man1/docker-volume-prune.1.gz
-rw-r--r-- root/root       333 2018-08-02 22:25 ./usr/share/man/man1/docker-volume-rm.1.gz
-rw-r--r-- root/root       557 2018-08-02 22:25 ./usr/share/man/man1/docker-volume.1.gz
-rw-r--r-- root/root       278 2018-08-02 22:25 ./usr/share/man/man1/docker-wait.1.gz
-rw-r--r-- root/root      1140 2018-08-02 22:25 ./usr/share/man/man1/docker.1.gz
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/man/man5/
-rw-r--r-- root/root      6342 2018-08-02 22:25 ./usr/share/man/man5/Dockerfile.5.gz
-rw-r--r-- root/root      1341 2018-08-02 22:25 ./usr/share/man/man5/docker-config-json.5.gz
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/man/man8/
-rw-r--r-- root/root     10525 2018-08-02 22:25 ./usr/share/man/man8/dockerd.8.gz
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/zsh/
drwxr-xr-x root/root         0 2018-08-02 22:25 ./usr/share/zsh/vendor-completions/
-rw-r--r-- root/root    125591 2018-08-01 20:07 ./usr/share/zsh/vendor-completions/_docker
```

</details>

<details>
<summary>Bits that are included in the new `docker-ce` package</summary>

```
drwxr-xr-x root/root         0 2018-08-02 22:31 ./
drwxr-xr-x root/root         0 2018-08-02 22:31 ./usr/
drwxr-xr-x root/root         0 2018-08-02 22:31 ./usr/share/
drwxr-xr-x root/root         0 2018-08-02 22:31 ./usr/share/doc/
drwxr-xr-x root/root         0 2018-08-02 22:31 ./usr/share/doc/docker-ce/
-rw-r--r-- root/root      1576 2018-06-11 18:14 ./usr/share/doc/docker-ce/README.md
-rw-r--r-- root/root       156 2018-08-02 22:31 ./usr/share/doc/docker-ce/changelog.Debian.gz
```

</details>

File: deb/common/control [hunk 2]
```diff
@@ -29,3 +29,28 @@ Description: Docker: the open-source application container engine
  language, framework or packaging system. That makes them great building blocks
  for deploying and scaling web apps, databases, and backend services without
  depending on a particular stack or provider.
+
+Source: docker-ce-cli
+Section: admin
+Priority: optional
+Maintainer: Docker <support@docker.com>
+Standards-Version: 3.9.6
+Homepage: https://dockerproject.org
```
Comment (seemethere) [inline deb/common/control]: Should change this to `https://docker.com`

File: deb/common/rules [hunk 3]
```diff
@@ -10,41 +10,21 @@ override_dh_gencontrol:
 	dh_gencontrol
 
 override_dh_auto_build:
-	cd engine && ./hack/make.sh dynbinary
 	cd /go/src/github.com/docker/cli && LDFLAGS='' make VERSION=$(VERSION) GITCOMMIT=$(DOCKER_GITCOMMIT) dynbinary manpages
 
 override_dh_auto_test:
```
Comment (seemethere) [inline deb/common/rules]: We could probably just remove this target altogether.

File: deb/common/rules [hunk 4]
```diff
@@ -10,41 +10,21 @@ override_dh_gencontrol:
...
-
-override_dh_installudev:
-	# match our existing priority
-	dh_installudev --priority=z80
+	mkdir -p debian/docker-ce-cli/usr/bin
+	cp -aTL cli/build/docker debian/docker-ce-cli/usr/bin/docker
 
 override_dh_install:
 	dh_install
+	# TODO Can we do this from within our container?
```
Comment (seemethere) [inline deb/common/rules]: possibly, but I think it'd have to be done in the postinst instead.

**Comment by seemethere** [comment]
Can confirm that the `docker-ce-cli` package is installable :+1:

**Comment by dhiltgen** [comment]
Comments covered in #130

**Comment by mitar** [comment]
It seems this PR also removes `/etc/init.d/docker` from both `docker-ce` and `docker-ce-cli` packages. This now broke our system because one cannot anymore run `service docker start` inside a Docker container to have docker-in-docker situation.

**Pull request merged**